diff --git a/dist/cloudflare/index.d.ts b/dist/cloudflare/index.d.ts
index 047dfb07b15d06f4daa6d297fc8512594f49152c..c8141286d55c94e0842f4d23db7cafe71fa07ccf 100644
--- a/dist/cloudflare/index.d.ts
+++ b/dist/cloudflare/index.d.ts
@@ -3,7 +3,7 @@ import { CloudflareEnvironment } from "alchemy/Cloudflare";
 import { RuntimeContext } from "alchemy/RuntimeContext";
 import { Self } from "alchemy/Self";
 import { Context, Effect, Layer, Scope } from "effect";
-import { HttpRouter } from "effect/unstable/http";
+import { HttpRouter } from "effect/http";
 import { ApplicationDefinition } from "@effront/core";
 import { HttpApplicationEffect } from "@effront/core/http";
 //#region src/cloudflare/index.d.ts
diff --git a/dist/cloudflare/index.js b/dist/cloudflare/index.js
index 5d94e1dd592b94d9971949743e6dc704c8732369..1899f0c0bab48ff1551126205518a9b566b989d8 100644
--- a/dist/cloudflare/index.js
+++ b/dist/cloudflare/index.js
@@ -3,7 +3,7 @@ import { CloudflareEnvironment } from "alchemy/Cloudflare";
 import { RuntimeContext } from "alchemy/RuntimeContext";
 import { Self } from "alchemy/Self";
 import { Context, Effect, Layer, Scope } from "effect";
-import { HttpRouter, HttpServerRequest } from "effect/unstable/http";
+import { HttpRouter, HttpServerRequest } from "effect/http";
 //#region src/cloudflare/index.ts
 const omitHostServices = Context.omit(RuntimeContext, Worker.Self, CloudflareEnvironment, Self, Layer.CurrentMemoMap, Request, WorkerEnvironment, WorkerExecutionContext, Scope.Scope, HttpServerRequest.HttpServerRequest, HttpServerRequest.ParsedSearchParams, HttpRouter.RouteContext, HttpRouter.HttpRouter);
 /**
diff --git a/package.json b/package.json
index 4b551b9bbd5c0dee2b25fbd7bdaf6a57511372f3..a89d6a8c507bd6cd1de3c764c6ea833b6def314a 100644
--- a/package.json
+++ b/package.json
@@ -30,18 +30,18 @@
     "tag": "latest"
   },
   "devDependencies": {
-    "@effect/platform-node": "4.0.0-rc.116",
+    "@effect/platform-node": "^4.0.1",
     "@effront/core": "0.2.0",
     "@effront/vite": "0.2.0",
-    "alchemy": "2.0.0-beta.79",
-    "effect": "4.0.0-rc.116",
+    "alchemy": "2.0.0-beta.81",
+    "effect": "^4.0.1",
     "vite": "npm:@voidzero-dev/vite-plus-core@0.3.1"
   },
   "peerDependencies": {
     "@effront/core": "^0.2.0",
     "@effront/vite": "^0.2.0",
-    "alchemy": "2.0.0-beta.79",
-    "effect": "4.0.0-rc.116",
+    "alchemy": "2.0.0-beta.81",
+    "effect": "^4.0.1",
     "vite": "*"
   }
 }
